packages/pds/src/mailer/moderation.ts

```diff
@@ -0,0 +1,36 @@
+import { Transporter } from 'nodemailer'
+import Mail from 'nodemailer/lib/mailer'
+import SMTPTransport from 'nodemailer/lib/smtp-transport'
+import { ServerConfig } from '../config'
+import { mailerLogger } from '../logger'
+
+export class ModerationMailer {
+  private config: ServerConfig
+  transporter: Transporter<SMTPTransport.SentMessageInfo>
+
+  constructor(
+    transporter: Transporter<SMTPTransport.SentMessageInfo>,
+    config: ServerConfig,
+  ) {
+    this.config = config
+    this.transporter = transporter
+  }
+
+  async send({ content }: { content: string }, mailOpts: Mail.Options) {
+    const res = await this.transporter.sendMail({
+      ...mailOpts,
+      text: content,
+      from:
+        mailOpts.from ??
+        (this.config.moderationEmailUser || this.config.emailNoReplyAddress),
```

Comment:
Is `moderationEmailUser` an email address, or just a username?

Reply:
(If it's just a username, I'm not sure if it can be used here.)

Reply:
It's an email address, that makes me think the name should probably reflect that. Would `moderationEmailAddress` have been clearer?

Reply:
Seem alright either way to me, just wanted to ensure it was an email address being used here 👍